EIPS/eip-5299.md

```diff
@@ -0,0 +1,168 @@
+---
+eip: 5299
+title: Layer3 scaling pattern
```

Comment:
While we normally don't curate EIP titles, I'm a bit uncomfortable with this title given how widely used the term `Layer 2` is in the Ethereum ecosystem, and calling this `Layer 3 Scaling` may imply to the reader that this is some kind of overarching design pattern we are working toward rather than a mechanism for dealing with contract storage.

Also, the name doesn't seem at all related to what this EIP actually purports to do (contract upgrade pattern).

Reply:
Thanks for your comments on our Ethereum Improvement Proposal. Everyone is entitled to their opinion irrespective of its relevance, or wisdom.

If you have specific feedback on the proposed design short-comings, such as probable oversights, or edge cases that the developers might have overseen, me & my team of developers are happy to look into it.

If you have trouble understanding the scale & scope of value such a simplistic looking design pattern could bring to the ecosystem, you can look at Uniswap v1,v2,v3 as an example of the world that's deprived of this EIP. If you still have trouble, i'd happy to connect with you over a call, or probably a twitter space even to dissect the proposal in greater detail.

Reply:
As editors, it is our job to ensure that content in new EIPs is appropriate content for an EIP.  We also work to ensure that EIPs follow fairly consistent standards and practices as much as possible.  While I can appreciate that many people think that the scope of EIPs should be broader than what it currently is (standards), at the moment this repository is primarily for standards, not design patterns and best practices.

Reply:
I agree with micah here.